sql/promscale--0.0.0.sql

```diff
@@ -372,6 +380,29 @@ END;
 $block$
 ;
 
+DO $block$
+DECLARE
+    _rec record;
+BEGIN
+    FOR _rec IN
+    (
+        SELECT m.*
+        FROM _prom_catalog.metric m
+        WHERE table_schema = 'prom_data'
+        ORDER BY m.table_schema, m.table_name
+    )
+    LOOP
+        EXECUTE format($sql$ALTER TABLE %I.%I OWNER TO %I$sql$, _rec.table_schema, _rec.table_name, current_user);
+        EXECUTE format($sql$GRANT TRIGGER ON TABLE %I.%I TO prom_modifier$sql$, _rec.table_schema, _rec.table_name);
```

Comment:
what's the reason for this grant? especially to prom_modifier?

Reply:
maybe we should actually remove trigger from line 286 in migration/idempotent/001-base.sql

Reply:
Trigger was needed to be able to create continuous aggregates if I'm not mistaken. Will take another look tomorrow